Add Send & ffsend to File Sharing page (#1837)
privacyguides/privacyguides.org: docs/file-sharing.en.md

!!! recommendation

    ![Send logo](assets/img/file-sharing-sync/send.svg){ align=right }

    **Send** is a fork of Mozilla’s discontinued Firefox Send service which allows you to send encrypted files to other users. Files are encrypted so that the server cannot read them, and they can be optionally password-protected as well. The maintainer of Send hosts a [public instance](https://send.vis.ee/). You can use other public instances, or you can host Send yourself.

    [:octicons-home-16: Repository](https://github.com/timvisee/send){ .md-button .md-button--primary }
    [:octicons-server-16:](https://github.com/timvisee/send-instances/){ .card-link title="Public Instances"}
    [:octicons-info-16:](https://github.com/timvisee/send#readme){ .card-link title=Documentation}
    [:octicons-code-16:](https://github.com/timvisee/send){ .card-link title="Source Code" }

Send can be used via its web interface or via [ffsend](https://github.com/timvisee/ffsend), a command-line client. If you are familiar with the command-line and send files frequently, we recommend using the native client to avoid JavaScript-based encryption. You can specify the --host flag to use a specific server:

> `ffsend upload --host https://send.vis.ee/ FILE`

Reviewer: I can't figure out how to make a GitHub suggestion with this change because it interferes with the Markdown syntax, but this should be a code block like with <code>```</code> above and below it.

Reviewer: ````suggestion
```
ffsend upload --host https://send.vis.ee/ FILE
```
````

## File Sharing

### Send

!!! recommendation

    ![Send logo](assets/img/file-sharing-sync/send.svg){ align=right }

    **Send** is a fork of Mozilla’s discontinued Firefox Send service which allows you to send files to others with a link. Files are encrypted on your device so that they cannot be read by the server, and they can be optionally password-protected as well. The maintainer of Send hosts a [public instance](https://send.vis.ee/). You can use other public instances, or you can host Send yourself.

    [:octicons-home-16: Repository](https://github.com/timvisee/send){ .md-button .md-button--primary }
    [:octicons-server-16:](https://github.com/timvisee/send-instances/){ .card-link title="Public Instances"}
    [:octicons-info-16:](https://github.com/timvisee/send#readme){ .card-link title=Documentation}
    [:octicons-code-16:](https://github.com/timvisee/send){ .card-link title="Source Code" }

> Send can be used via its web interface or via [ffsend](https://github.com/timvisee/ffsend), a command-line client. If you are familiar with the command-line and send files frequently, we recommend using the native client to avoid JavaScript-based encryption. You can specify the --host flag to use a specific server:

Reviewer 2: ```suggestion
Send can be used via its web interface or via [ffsend](https://github.com/timvisee/ffsend), a CLI. 
```

There's an abbreviation for "CLI" in abbreviations.en.md, so it could be worth just shortening this (unless that shifts the meaning too much)?

Reviewer: https://github.com/privacyguides/privacyguides.org/pull/1837#discussion_r995893881

Reviewer 3: Hmm, i think removing this warning would be a bad idea.

Maybe an admonishment would be best.

> Send can be used via its web interface or via [ffsend](https://github.com/timvisee/ffsend), a command-line client. If you are familiar with the command-line and send files frequently, we recommend using the native client to avoid JavaScript-based encryption. You can specify the `--host` flag to use a specific server:
> 
> ```text

Reviewer: Instead of https://github.com/privacyguides/privacyguides.org/pull/1837#discussion_r995448697? Also pretty sure this should be `bash` not `text` as a shell command.

````suggestion
Send can be used via its web interface or via the [ffsend](https://github.com/timvisee/ffsend) CLI. If you are familiar with the command-line and send files frequently, we recommend using the CLI client to avoid JavaScript-based encryption. You can specify the `--host` flag to use a specific server:

```bash
````

    ![Send logo](assets/img/file-sharing-sync/send.svg){ align=right }

>      **Send** is a fork of Mozilla’s discontinued Firefox Send service which allows you to send files to others with a link. Files are encrypted on your device so that they cannot be read by the server, and they can be optionally password-protected as well. The maintainer of Send hosts a [public instance](https://send.vis.ee/). You can use other public instances, or you can host Send yourself.

Reviewer: ```suggestion
    **Send** is a fork of Mozilla’s discontinued Firefox Send service which allows you to send files to others with a link. Files are encrypted on your device so that they cannot be read by the server, and they can be optionally password-protected as well. The maintainer of Send hosts a [public instance](https://send.vis.ee/). You can use other public instances, or you can host Send yourself.
```
